From bf7cadfd2165c05177f9c618e183d956c748bc84 Mon Sep 17 00:00:00 2001
From: Ewan Milne <emilne@redhat.com>
Date: Wed, 20 Aug 2014 18:59:30 -0400
Subject: [scsi] scsi_debug: make pseudo_primary static

Message-id: <1408561176-18133-52-git-send-email-emilne@redhat.com>
Patchwork-id: 88452
O-Subject: [RHEL7.1 PATCH e-stor 51/57] [SCSI] scsi_debug: make pseudo_primary static
Bugzilla: 1132100
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: "Ewan D. Milne" <emilne@redhat.com>

upstream commit 11ddceca99ae9a3e124b91dfe9c5342ee09f376d

Tested by Ewan D. Milne and Barry Donahue with a combination
of manual and automated testing.  No issues seen.

upstream commit 11ddceca99ae9a3e124b91dfe9c5342ee09f376d
Author: Akinobu Mita <akinobu.mita@gmail.com>
Date:   Wed Feb 26 22:56:59 2014 +0900

    [SCSI] scsi_debug: make pseudo_primary static

    As pseudo_primary is only used in scsi_debug.c, it should be static.

    Signed-off-by: Akinobu Mita <akinobu.mita@gmail.com>
    Reviewed-by: Douglas Gilbert <dgilbert@interlog.com>
    Reviewed-by: Martin K. Petersen <martin.petersen@oracle.com>
    Signed-off-by: James Bottomley <JBottomley@Parallels.com>

Signed-off-by: Ewan D. Milne <emilne@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/scsi_debug.c b/drivers/scsi/scsi_debug.c
index bdfb9be..9cd211e 100644
--- a/drivers/scsi/scsi_debug.c
+++ b/drivers/scsi/scsi_debug.c
@@ -3246,7 +3246,7 @@ static struct attribute *sdebug_drv_attrs[] = {
 };
 ATTRIBUTE_GROUPS(sdebug_drv);
 
-struct device *pseudo_primary;
+static struct device *pseudo_primary;
 
 static int __init scsi_debug_init(void)
 {
-- 
1.7.1